sensei-lms.php

```diff
@@ -3,7 +3,7 @@
  * Plugin Name: Sensei LMS
  * Plugin URI: https://woocommerce.com/products/sensei/
```

Comment:
Just another small detail that I noticed just now...WDYT about changing this to the senseilms.com website?

Right now the redirect is already there, but I guess it's worth changing:

```suggestion
 * Plugin URI: https://senseilms.com/
```

Reply:
Good catch! Just to be sure, @burtrw is there some reason why we didn't change that? Or did we just miss it?

Reply:
I'm going to go ahead and merge without this change. Let's add it in a new PR and discuss, just to be sure.

Reply:
No reason that I remember, happy to point to our home page :)

Reply:
Just for documentation purposes, I sent a PR to fix this small detail: https://github.com/Automattic/sensei/pull/5287